src/nimblepkg/cli.nim

```diff
@@ -169,20 +175,124 @@ proc promptCustom*(forcePrompts: ForcePrompt, question, default: string): string
 proc promptCustom*(question, default: string): string =
   return promptCustom(dontForcePrompt, question, default)
 
+proc getKey(): Key =
+  ## Needed in order to allow arrow key control for Windows. It also makes it
+  ## easier to determine which keys are pressed on POSIX systems. This is based
+  ## of terminal.getch for the standard library.
```

Comment:
Why can't this be used directly?

Reply:
I initially tried that but under Windows `getch` returns only `0` with the arrow keys (works find for enter and space). The standard library returns `keyEvent.uChar` which doesn't appear to hold an actual value when the arrow keys are pressed. Using `getch` alone would prevent Windows users from selecting with the arrow keys. Could use the space to cycle through items on Windows. I thought about changing/adding to `terminal`, but I'm unsure what would be the best way forward.

Reply:
The best thing would be to patch the stdlib I think for this case.

Reply:
But that can be separately if you wish.

Reply:
I think you're right. I submitted nim-lang/nim#6966 to attempt to fix it. If it gets accepted `getKey` may still be useful but can be removed along with the Key enum.